src/Modal/index.tsx

```diff
@@ -16,7 +45,18 @@ interface Props {
 }
 
 const Modal: FunctionComponent<Props> = ({
-  header = '',
+  header = {
```

Comment:
For "header" and "subheader", let's force their styles and not have them adjustable. So I would just make this prop a string. The style shouldn't change per modal. If it does, we will address the problem then. So make the header variant an "h4" with "center" alignment and not have it customizable for now

Reply:
Roger that. I'll update to make the styles both forced and non-adjustable. For the header, a variant of "h4" and "center" alignment. For the subheader, I was planning to make its variant "paragraph" and "center" alignment. Does that sound good?

Reply:
yes!